src/usd/sdf_usd_parser/sensor.cc

```diff
@@ -124,7 +127,17 @@ namespace usd
     }
 
     if (typeParsed)
-      usd::SetPose(_sensor.RawPose(), _stage, sdfSensorPath);
+    {
+      if (_sensor.Type() == sdf::SensorType::CAMERA)
+      {
+        ignition::math::Pose3d poseCamera(0, 0, 0, 1.57, 0, -1.57);
+        usd::SetPose(poseCamera * _sensor.RawPose(), _stage, sdfSensorPath);
```

Comment:
I just read the following note in the [UsdGeomCamera docs](https://graphics.pixar.com/usd/release/api/class_usd_geom_camera.html#details):
```
Cameras in USD are always "Y up", regardless of the stage's orientation (i.e. UsdGeomGetStageUpAxis()).
This means that the inverse of 'camXform' (the VIEW half of the MODELVIEW transform in OpenGL parlance) will transform the world such that the camera is at the origin, looking down the -Z axis, with +Y as the up axis, and +X pointing to the right.
This describes a right handed coordinate system.
```

So, are you applying the rotations here to fix the default camera direction? If so, that makes sense, but would you mind adding a comment that explains why this has to be done? You can even reference the link I just included above in the comment if that's easiest